### PR TITLE
Add-To-Cart

### PR DESCRIPTION
**Pull request by richarddubay**

### CODE REVIEW

**Comment by jbaxleyiii** [comment]
**[Storybook Hub](https://hub.getstorybook.io/)** is building storybooks for every commit in this PR.

**Latest Storybook: https://42dcc2c1-edaf-4d42-8b65-9aba9c612035.sbook.io/**
<sub>( Visit [here](https://hub.getstorybook.io/app/github.com/NewSpring/Holtzman/branch/add-to-cart/) for more info )</sub>

> _This comment is automatically added by Storybook Hub_

File: imports/blocks/checkout-buttons/index.js [hunk 1]
```diff
@@ -188,8 +195,4 @@ class CheckoutButtons extends Component {
   }
 }
 
-export default withSavedPayments(withState(CheckoutButtons));
-
-export {
-  CheckoutButtons as CheckoutButtonsWithoutData,
-};
```
Comment (richarddubay) [inline imports/blocks/checkout-buttons/index.js]: @jbaxleyiii if we don't do this here, then we run into the problem that we were seeing with the linter not liking the import of the component in tests (SubFund was the example you and I talked about the other day) and then we have to disable linting on that line. We can't have two exports named the same thing, from a linting point of view. Is there a different way to solve the linting problem?

**Comment by jbaxleyiii** [comment]
@richarddubay https://github.com/NewSpring/Holtzman/pull/1455 and https://350f2695-2355-4f47-a4a3-642cbaa2fa59.sbook.io/?knob-Fund=General%20Fund&selectedKind=Add%20To%20Cart&selectedStory=With%20one%20account&full=0&down=1&left=1&panelRight=1&downPanel=storybook-addon-background%2Fbackground-panel

**Comment by richarddubay** [comment]
@jbaxleyiii LOVE. IT.

**Comment by richarddubay** [comment]
@jbaxleyiii The undefined bits in the snapshots should be fixed now.

**Comment by richarddubay** [comment]
@jbaxleyiii I removed the WIP tag from this PR. We could make a couple more changes probably, but I think it can be done for now.

**Pull request merged**